IBPSA/Resources/C-Sources/WeeklySchedule.h

```diff
@@ -0,0 +1,67 @@
+/*
+
+  This code implements a weekly schedule. Input text example:
+
+#some comment
+double  tab1(3,5) # another comment
+# blabla
+mon:0:0:10              -       3       1       -
+tue,thu:20:30:59        123     -       45      -
+wed,sat,sun             12      1       4       -
+
+
+	License: BSD3
```

Comment:
Why specifying a license rather than using the top-level license of the library?

Reply:
I copied this since MSL does it. I think it's useful in case someone copies the code into a separate library.